content/04.results.md

```diff
@@ -1 +1,195 @@
+
+### Celtic English Names are over enriched in cited and quoted persons, while East Asian Names are under enriched
+
+![
+**Analysis of Quotes and Citations found Overrepresentation of Celtic/English and underrepresentation of East Asian names**
+Panel A, left, depicts an example of the names extracted from quoted speakers and citations found within news articles and authors in research articles.
+Panel A, right, highlights the data types and processes used to analyze the origin of extracted names.
+Panels B and D depicts nine trend lines, each representing a possible name origin.
+Panels C and E depict two possible trend lines, comparing either cited last authors or quoted speakers against last authors of Nature research articles.
+We observed that the overrepresentation of Celtic/English names is present both in citations and quotes.
+Furthermore, the overrepresentation of East Asian names is also present both in citations and quotes.
+We recapitulate this finding when comparing against an additional background of Springer published last authors (Supplemental Figure XX)
+](https://github.com/nrosed/nature_news_disparities/raw/09df81eac6d8450127552f0b9dd3a6c1212b10eb/figure_notebooks/tmp_files/fig3_tmp/fig3_main.png "Figure 3"){#fig:fig3 width=6in}
+
+
+To identify possible bias in name origin, we again used the extracted quoted speakers and last authors published in Nature.
+In addition, we also identified the last authors of all articles cited by a news-related article.
+All processed names were then inputted into Wiki2019-LSTM and used to assign one of nine possible name origins ([Methods]).
+Figure @fig:fig3 a displays a schematic of the data processing and example names used in this analysis.
+We divided our analysis into two parts: firstly, looking at the number of cited last authors with a specific name origin overall cited last authors in a particular year.
+Secondly, looking at the proportion of quotes from a speaker with a specific name origin overall quotes in a particular year.
+We used the proportion of last authors in Nature with a specific name origin overall Nature articles in a particular year for both analyses.
+Additionally, in our supplemental analyses, we compare against the last authorship in a random selection of Springer articles.
+We find that the number of quotes dramatically outnumbers the number of authors and citations (Supplemental Figure 3 a).
+Still, the total number of observations per data type is larger than [XX] across all years.
+
+In comparing the citation rate of last author name origins in news articles, we find most names are Celtic/English or European, both with a citation rate between 30-40% [MORE SPECIFIC] (Figure @fig:fig3 b).
+East Asian names are the highest proportion of cited names, with 10-20% cited last authorships [MORE SPECIFIC].
```

Comment:
I don't understand how this aligns with the sentence above it.

Reply:
oops, missed a word. Fixed here: https://github.com/greenelab/nature_news_manuscript/pull/1/commits/61eb711d02938c47fee285c28a1a3d7ee005e00a